config/initializers/govuk_notify.rb

```diff
@@ -43,6 +45,8 @@
   consent_confirmation_refused: "eb34f3ab-0c58-4e56-b6b1-2c179270dfc3",
   consent_school_reminder: "ee3d36b1-4682-4eb0-a74a-7e0f6c9d0598",
   consent_school_request: "c7bd8150-d09e-4607-817d-db75c9a6a966",
+  unknown_consent_contact_details_warning:
+    "1fd4620d-1c96-4af1-b047-ed13a90b0f44",
```

Comment:
I think it would be good to keep this list in alphabetical order, and I wonder if we could name this with the `consent_` prefix so it goes with the others, something like:

```
consent_unknown_contact_details_warning
```